core/commands/commands_test.go

```diff
@@ -109,6 +109,7 @@ func TestCommands(t *testing.T) {
 		"/key/sign",
 		"/key/verify",
 		"/log",
+		"/log/get-level",
```

Comment:
I missed this. Could you provide a brief explanation of what adding this line does?